src/libs/utils.js

```diff
@@ -2,6 +2,13 @@ import select from 'select-dom';
 import elementReady from 'element-ready';
 import domLoaded from 'dom-loaded';
 
+/**
+ * Prevent fn's errors from blocking the remaining tasks.
+ * https://github.com/sindresorhus/refined-github/issues/678
+ * The code looks weird but it's synchronous and fn is called without args.
+ */
+export const safely = fn => new Promise(resolve => resolve(fn()));
```

Comment:
```js
export const safely = async fn => fn();
```

<img width="153" alt="screen shot 2017-10-17 at 13 42 18" src="https://user-images.githubusercontent.com/170270/31650323-06f59ee6-b341-11e7-83bb-2e1e65cd9066.png">

Reply:
This would have been much nicer with decorators:

```js
@safely
addYoursMenuItem();
```

Reply:
We have no decorators but this is also an option 😋 

```js
async () =>
addYoursMenuItem();
```